Client/src/app/artist-details/artist-details.component.html

```diff
@@ -0,0 +1,77 @@
+<div class="card mb-3">
+  <div class="row g-0">
+    <div class="col-md-4">
+      <img
+        src="{{ artist.imageUrl }}"
+        class="img-fluid rounded-start"
+        alt="{{ artist.name }}"
+      />
+    </div>
+    <div class="col-md-8">
+      <div class="card-body">
+        <h5 class="card-title fs-4">{{ artist.name }}</h5>
+      </div>
+    </div>
+  </div>
+</div>
+
+<ul class="nav nav-tabs">
+  <li class="nav-item" (click)="onOverviewSelect()">
+    <a
+      class="nav-link"
+      routerLink="/{{ componentRouteName }}/{{ artist.name }}"
+      routerLinkActive="active"
+      [routerLinkActiveOptions]="{ exact: true }"
+      >Overview
+    </a>
+  </li>
+  <li class="nav-item" (click)="onTopTracksSelect()">
+    <a
+      class="nav-link"
+      routerLink="/{{ componentRouteName }}/{{ artist.name }}/{{
```

Comment:
Formatting seems strange. Is it something like Prettier or did you do it on purpose?